src/api/http-common.ts

```diff
@@ -1,5 +1,10 @@
 import axios, { AxiosInstance, AxiosResponse } from "axios";
 
+interface ValidationError {
```

Comment:
Am I being McThickThick? Where is this used?

Reply:
You certainly aren't - it is a legacy of me attempting to cast the axios error which I learnt is not a good idea